--- dumb-1.0/CMakeLists.txt.omv~	2016-10-28 12:29:53.392482722 +0200
+++ dumb-1.0/CMakeLists.txt	2016-10-28 12:29:59.667334274 +0200
@@ -143,7 +143,7 @@ if(BUILD_ALLEGRO4)
     add_library(aldmb ${ALLEGRO_SOURCES})
     list(APPEND DUMB_TARGETS aldmb)
     list(APPEND INSTALL_HEADERS include/aldumb.h)
-    target_link_libraries(aldmb ${ALLEGRO_LIBRARIES})
+    target_link_libraries(aldmb ${ALLEGRO_LIBRARIES} dumb)
 endif()
 
 if(BUILD_EXAMPLES)
